### ISSUE TITLE
Navigator Error

### ISSUE THREAD

**Issue by swernofsky** [ContinuumIO/anaconda-issues#3059]
## Main error Application  b>spyder /b> launch may have produced errors. ## Traceback      warn  kq_init: detected broken kqueue; not using.: Undefined error: 0  warn  kq_init: detected broken kqueue; not using.: Undefined error: 0  warn  kq_init: detected broken kqueue; not using.: Undefined error: 0  warn  kq_init: detected broken kqueue; not using.: Undefined error: 0  warn  kq_init: detected broken kqueue; not using.: Undefined error: 0  warn  kq_init: detected broken kqueue; not using.: Undefined error: 0  warn  kq_init: detected broken kqueue; not using.: Undefined error: 0  warn  kq_init: detected broken kqueue; not using.: Undefined error: 0     ## System information     python: 3.6.1 language: en os: Darwin;16.6.0;Darwin Kernel Version 16.6.0: Fri Apr 14 16:21:16 PDT 2017; root:xnu-3789.60.24~6/RELEASE_X86_64;x86_64;i386 version: 1.6.2 platform: osx-64 qt: 5.6.2 pyqt: 5.6.0 conda: 4.3.21

**Comment by swernofsky** [comment]
Jupyter also failed to launch. This was after failing to install with the graphic installer, and using the terminal installer. Perhaps my environment is messed up?

**Comment by goanpeca** [comment]
Closing as duplicate of #1778

Please update to the latest version of Navigator.

On the terminal type

```
$ conda update anaconda-navigator
```

**Comment by swernofsky** [comment]
Hi!

Thanks, this was really helpful. I also had a question about Jupyter not
starting properly after I installed Anaconda. Is this a separate bug, or do
you have a quick solution?

1. I tried to install Anaconda on MacOS 10.12.5 from the graphical
installer, but it said that it "could not install Anaconda in this
location". Yes, this was an administrator account.

2. I tried a 2nd time with the terminal installer. This worked fine.

3. Per your instructions, I ran "conda update anaconda-navigator" from the
terminal. This worked fine with Spider3.

4. I also ran "conda update conda" and "conda update anaconda", both of
which told me there was no need to update anything.

5. I tried to launch Jupyter 5.0.0 from Anaconda, and got this message on
terminal:

Last login: Tue Jul 11 20:52:20 on ttys000
/Users/admin/anaconda3/bin/jupyter_mac.command ; exit;
London-Below:~ admin$ /Users/admin/anaconda3/bin/jupyter_mac.command ; exit;
[I 12:16:07.657 NotebookApp] Serving notebooks from local directory:
/Users/admin
[I 12:16:07.657 NotebookApp] 0 active kernels
[I 12:16:07.657 NotebookApp] The Jupyter Notebook is running at:
http://localhost:8888/?token=83da52349f0c3d2f2314eb5a0358f24094ff6605928dbd98
[I 12:16:07.657 NotebookApp] Use Control-C to stop this server and shut
down all kernels (twice to skip confirmation).
[C 12:16:07.658 NotebookApp]

    Copy/paste this URL into your browser when you connect for the first
time,
    to login with a token:

http://localhost:8888/?token=83da52349f0c3d2f2314eb5a0358f24094ff6605928dbd98
0:97: execution error: "
http://localhost:8888/tree?token=4abde0b9fbe0cb89bd579fedcb371723b82e2fd42d42386c"
doesn’t understand the “open location” message. (-1708)

6. I was able to connect my browser (most recent version of Firefox) to
http://localhost:8888/?token=83da52349f0c3d2f2314eb5a0358f24094ff6605928dbd98,
and was presented with a Jupyter page showing my file system.

7. I was unable (or just don't know how) to get a Jupyter notebook to start
from there.

How do I fix this?

Thanks in advance.
Steve

Steven A. Swernofsky, Esq.
(650) 518-5080 office, (650) 799-2438 mobile
